packages/apps/contracts/HighRollerApp.sol

```diff
@@ -120,59 +120,39 @@ contract HighRollerApp is CounterfactualApp {
     return abi.encode(nextState);
   }
 
-  function resolve(bytes calldata encodedState, Transfer.Terms calldata terms)
+  function resolve(bytes calldata encodedState)
     external
     pure
-    returns (Transfer.Transaction memory)
+    returns (TwoPartyEthAsLump.Resolution)
```

Comment:
Shouldn't this include `memory`?

Reply:
this is fixed to be `bytes memory` but in general, "simple" return types are not annotated with `memory`